templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -683,6 +683,7 @@ export class {{ service.name }}Client {
   }
 {%- endfor %}
 {%- for method in service.paging %}
+  {%- if not method.pagingMapResponseType %}
```

Comment:
nit: please align `if` and its corresponding `endif` (we tend not to offset nested template instructions to the right)

Reply:
sg